tests/run.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python
+
+from sys import exit
+
+
+import dj_database_url
+import django
+from colour_runner.django_runner import ColourRunnerMixin
+from django.conf import settings
+from django.test.runner import DiscoverRunner
+
+
+settings.configure(
+    ROOT_URLCONF='articles.urls',
+    DATABASES={
+        'default': dj_database_url.config(
+            default='postgres://localhost/articles'
+        ),
+    },
+    INSTALLED_APPS=(
+        'articles',
+
+        'feincms',
+
+        'django.contrib.contenttypes',
+    ),
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'APP_DIRS': True,
+        }
+    ],
+)
```

Comment:
It's often worth adding:

```
PASSWORD_HASHERS=('django.contrib.auth.hashers.MD5PasswordHasher',)
```
to speed things up a little when it comes to tests involving users (although I realise you don't have any yet, so perhaps it's premature.